debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/logminer/LogMinerStreamingChangeEventSource.java

```diff
@@ -680,7 +680,7 @@ public void endMiningSession(OracleConnection connection, OracleOffsetContext of
     }
 
     private void updateBatchSize(boolean increment) {
-        if (increment && currentBatchSize < connectorConfig.getLogMiningBatchSizeMin()) {
+        if (increment && currentBatchSize < connectorConfig.getLogMiningBatchSizeMax()) {
             currentBatchSize += connectorConfig.getLogMiningBatchSizeMin();
```

Comment:
```suggestion
        if (increment && currentBatchSize < connectorConfig.getLogMiningBatchSizeMax()) {
            currentBatchSize += connectorConfig.getLogMiningBatchSizeMin();
```
I suggest we also use `Math.min` on the increment line and insure that the batch size never exceeds the max.  This guarantees that the log entry is always processed properly. 

Additionally, I'd look at the `else if` condition for `< LogMiningBatchSize` where we decrement the batch size and likely use `Math.max` to handle the same but opposite scenario where the `currentBatchSize` never falls below the min.